joblib/_store_backends.py

```diff
@@ -189,6 +190,8 @@ def write_func(to_write, dest_filename):
                                       compress=self.compress)
 
             self._concurrency_safe_write(item, filename, write_func)
+        except PicklingError:
```

Comment:
This `try/except` block should be in the `write_func` function.

Reply:
Done